internal/apitesting/client.go

```diff
@@ -184,3 +190,65 @@ func (c *Client) composeURL(path string, buildID string, format string, tunnel c
 
 	return url.String()
 }
+
+// GetProjects returns the list of Project available.
+func (c *Client) GetProjects(ctx context.Context) ([]Project, error) {
+	url := fmt.Sprintf("%s/api-testing/api/project", c.URL)
+	req, err := requesth.NewWithContext(ctx, http.MethodGet, url, nil)
+	if err != nil {
+		return []Project{}, err
+	}
+
+	req.SetBasicAuth(c.Username, c.AccessKey)
+	resp, err := c.HTTPClient.Do(req)
+	if err != nil {
+		return []Project{}, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode >= http.StatusInternalServerError {
+		return []Project{}, errors.New(msg.InternalServerError)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		body, _ := io.ReadAll(resp.Body)
+		return []Project{}, fmt.Errorf("request failed; unexpected response code:'%d', msg:'%v'", resp.StatusCode, string(body))
```

Comment:
Same result but less wrappers:
```suggestion
		return []Project{}, fmt.Errorf("request failed; unexpected response code:'%d', msg:'%s'", resp.StatusCode, body)
```